aws/table_aws_sagemaker_notebook_instance.go

```diff
@@ -276,7 +276,7 @@ func listAwsSageMakerNotebookInstanceTags(ctx context.Context, d *plugin.QueryDa
 
 func getAwsSageMakerNotebookInstanceTurbotTags(ctx context.Context, d *transform.TransformData) (interface{},
 	error) {
-		data := d.HydrateItem.(*sagemaker.ListTagsOutput)
+	data := d.HydrateItem.(*sagemaker.ListTagsOutput)
 
 	if data.Tags == nil {
```

Comment:
Do we need this block, as we returning nil at the end?
https://github.com/turbot/steampipe-plugin-aws/blob/f2b280097bc5e11414422ec75eea864f740c9eaa/aws/table_aws_sagemaker_notebook_instance.go#L281-L283